### ISSUE TITLE
Infinite money from EFTPOS

### ISSUE THREAD

**Issue by lmas** [Apollo-Community/ApolloStation#70]
Something something... Kai.H found out about it, might need to investigate it some more.

**Comment by Kwask** [comment]
Waiting to hear how it happens before we can really do anything about it

**Comment by lmas** [comment]
Yeah I know, it was just something I heard being mentioned on teamspeak.

**Comment by JMMJ01** [comment]
So what you're saying is, next canon round I should figure this out and
become a millionaire ICly from robbing NT :D

**Comment by Kai-Holmes** [comment]
I set up an account specifically to comment on this. For those interested I am the only known person with knowledge of this. I would just like to inform you that it is only possible for a HoP to perform such an act and because, in theory there is only a finite amount of money on station accounts. It is a valuable asset for any budding HoP such as myself. Particularly for things such as employment funds.

**Comment by lmas** [comment]
So you couldn't get infinite money then? You were just stealing from the station accounts? :p

**Comment by Kai-Holmes** [comment]
What I'm saying is that, the station accounts are finite and not designed for what I refer to as an 'employment fund'. This fund is used to make the prospect of changing career more desirable with the cash incentive. In RP it could be perceived that this is a NT approved scheme which has an account number attached to it so that HoPs can pull money from it as an enticement. In all reality I accessed the Financial Accounts Database and created a new account with somewhere in the region of $50000. I then assigned an ID card that account and would get transferring crew to enter their details into the EFTPOS scanner, set it to around $500-$1000 then scan the ID card. In theory, yes, infinite money. In reality, only possible for white listed HoPs.
